docs/development_suite/set-up-infrastructure/enhanced-project-workflow.md

```diff
@@ -0,0 +1,136 @@
+
+Moreover, in the new workflow the user will be able to **export Kubernetes configurations** by clicking on a download button.
+:::note  
+The review and export features are still in development and will be available in future releases.
+:::
+
+:::caution
+When deploying your configurations to the cluster, keep in mind that only the changes pushed to the repository’s default branch will be considered.
+:::
+
+The new deployment workflow also unlocks the possibility to adopt a **pull-based deployment strategy**, in which a GitOps tool is set up to automatically align the cluster’s state to the latest changes pushed to the Git repository. To find out more about pull-based deployment and its advantages, visit the [GitOps deployment strategy documentation page](/development_suite/deploy/gitops-based/index.md).
+
+## Activating the feature
+
+The GitOps support feature can be activated in different ways, depending on your needs. In order to access the feature, please request the support of a Mia-Platform referent.
+
+You can activate the `ENABLE_CONF_GENERATION_ON_DEPLOY` feature toggle on a new project or Company to enable the enhanced project workflow. The feature toggle activates the new versioning system and the generation of Kubernetes configurations at the time of deployment. You can enable it for a single project or for an entire Company, which means all projects in the Company will adopt the new workflow.
+
+If you already have a Company with some projects, and you do not wish to migrate them to the new approach just yet, you can activate the `ENABLE_CREATE_PROJECT_WITH_SAVE_CONFIG_ON_DEPLOY` feature toggle on the Company. This feature toggle makes sure that all new projects in the Company will be created with the new workflow, while leaving the existing ones untouched. 
+
+:::note
+If both feature toggles are enabled, `ENABLE_CONF_GENERATION_ON_DEPLOY` will prevail.
+:::
+
+:::info
+In order to set up a new project, follow the [project creation guide](/development_suite/set-up-infrastructure/create-project.mdx). To create a new Company or to activate feature toggles, please open a Service Request.
+:::
+
+## Manual adjustments on newly created projects
+
+After creating a new project and activating the feature toggles, there are some important manual adjustments necessary to make sure the project works correctly.
+
+First of all, the project model **must have set a default revision**: ask your Mia-Platform referent to check it, and in case it is missing, to add it manually. You can choose a custom name for your default revision, e.g. `main`.
+
+If you want the project to use a **pull-based deployment strategy**,<!-- TODO: (LINK) --> its configuration must contain the `strategy` property set to `pull` (the value for push-based deployments is `push`). Based on the selected strategy, the `runnerTool` may take different values. 
```

Comment:
Where this 'strategy' property is findable? I think we have to clarify better this paragraph

Reply:
the strategy is to be set from CMS (as of now but it will land in the project settings so let's add this to the roadmap paragraph).

the strategy can be found in `project.deploy.strategy` for the global configuration that is inherited by all environments; if you want an "environment specific settings" you can patch the `project.environments[_].deploy.strategy` (NB: this won't land in the project settings for now)